### DISCUSSION TITLE
Should I report obviously wrong routes?

### DISCUSSION THREAD

**Discussion by JonasVautherin** [valhalla/valhalla#4753] [Q&A] [answered]
I was playing with the Valhalla demo, and it plans weird routes when choosing the "Bike" or "Walk" profiles.

Is that an issue in Valhalla, in the profile, or in OSM? Is there something I can do to help get that better?

![20240606_21h12m34s_grim](https://github.com/valhalla/valhalla/assets/2606672/1763c24a-1048-4519-87dc-4960d1ffbad9)

EDIT: Actually I realise that Organic Maps generate the same kind of biking routes through those lakes, for some reason :thinking:.

**Comment by chrstnbwnkl** [comment, marked as answer]
"wrong" is always a bit of a harsh word :sweat_smile: For bike/ped, `use_ferry` defaults to `1.0`, i.e. the max value, if I'm not mistaken. And this might be sensible: strictly, using all these ferries would get you to your destination faster than merely walking/biking. If you turn that willingness down, you will get routes more akin to those you would want if you were building e.g. an outdoor sports route planner.

In more general terms, though, of course it makes sense to share routes that seem off. In those cases there are some things that make it easier to spot what may be going on:

  1. find a minimal, reproducible example 
  2. along with the screenshot, post the request as text
  3.  make sure you're using latest master (the [public instance](https://valhalla.openstreetmap.de/) usually is up to date)

**Reply by JonasVautherin** [reply]
Oh, that makes sense. I also realised that Organic Maps allows me to disable ferries. I guess I just need to find how to do that with Valhalla!

Thanks for the insights!

**Reply by JonasVautherin** [reply]
Found it! https://valhalla.github.io/valhalla/api/turn-by-turn/api-reference/#bicycle-costing-options
